packages/transport-bluetooth/src/client/trezor-bluetooth.ts

```diff
@@ -0,0 +1,75 @@
+import { WebsocketClient } from '@trezor/websocket-client';
+
+import {
+    BluetoothDevice,
+    BluetoothInfo,
+    Logger,
+    NotificationEvent,
+    TrezorBluetoothSettings,
+} from './types';
+
+// Client for trezor-bluetooth rust websocket server
+export class TrezorBluetooth extends WebsocketClient<NotificationEvent> {
+    readonly settings: TrezorBluetoothSettings;
+    readonly logger: Logger;
+
+    constructor(settings: TrezorBluetoothSettings) {
+        super({
+            url: settings.url,
+            timeout: settings.timeout || 60 * 1000,
+        });
+        this.settings = Object.freeze(settings);
+        this.logger = settings.logger || {
+            info: () => {},
+            debug: () => {},
+            log: () => {},
+            warn: () => {},
+            error: () => {},
+        };
+    }
+
+    createWebsocket() {
+        return this.initWebsocket({
+            url: this.settings.url,
+            headers: {
+                Origin: 'https://node.trezor.io',
```

Comment:
fyi - i have this commit in another branch https://github.com/trezor/trezor-suite/pull/18077/commits/dfdab84b21e7d336ca7190c462f8e6f5277c2af0 
what do you think?

Reply:
cool :+1:  then i will remove it from here

Reply:
18223

Reply:
already merged